pkg/volume/gcepd/attacher_test.go

```diff
@@ -379,34 +551,63 @@ func (testcase *testcase) DetachDisk(devicePath string, nodeName types.NodeName)
 func (testcase *testcase) DiskIsAttached(diskName string, nodeName types.NodeName) (bool, error) {
 	expected := &testcase.diskIsAttached
 
-	if expected.diskName == "" && expected.nodeName == "" {
-		// testcase.diskIsAttached looks uninitialized, test did not expect to
+	found := false
+	var err error = nil
+
+	if expected.attachedDisks == nil {
+		// testcase.attachedDisks looks uninitialized, test did not expect to
 		// call DiskIsAttached
 		testcase.t.Errorf("Unexpected DiskIsAttached call!")
```

Comment:
do we need to throw an error here if we're returning the error (I assume the caller calls `t.Errorf` at some point on this error. (realize this is not your code but might be worth just doing the quick fix for)

Reply:
ack

Reply:
I remove the unnecessary errorfs